js/modules/Debug/Debug.js

```diff
@@ -0,0 +1,1380 @@
+                      <thead>
+                      <tr>
+                         ${filtered_request_id === undefined ? `<th>${_x('debug', 'Request ID')}</th>` : ''}
+                         <th>${_x('debug', 'Number')}</th><th>${_x('debug', 'Query')}</th><th>${_x('debug', 'Time')}</th><th>${_x('debug', 'Rows')}</th><th>${_x('debug', 'Warnings')}</th><th>${_x('debug', 'Errors')}</th>
+                      </tr>
+                      </thead>
+                      <tbody></tbody>
+                   </table>
+                </div>
+            `);
+            initSortableTable('debug-sql-request-table');
+        }
+
+        const sql_table = content_area.find('table').first();
+        const sql_table_body = sql_table.find('tbody').first();
+
+        // get all the request IDs present in the SQL data (first column values)
+        let request_ids_present = new Set();
+        sql_table_body.find('tr td:first-child').each((index, cell) => {
+            request_ids_present.add($(cell).text());
+        });
+
+        const sql_data = this.getCombinedSQLData();
+
+        let rows_to_append = '';
+        $.each(sql_data['queries'], (request_id, queries) => {
+            if (request_ids_present.has(request_id) ||
+                (filtered_request_id !== undefined && filtered_request_id !== request_id)) {
+                return;
+            }
+            queries.forEach((query) => {
+                //Note: keep the query cell as a single line, or it will ruin the formatting of the contents
+                rows_to_append += `
+                    <tr>
+                        ${filtered_request_id === undefined ? `<td><button class="btn btn-link request-link">${request_id}</button></td>` : ''}
+                        <td>${query['num']}</td>
+                        <td style="max-width: 50vw; white-space: break-spaces;"><code class="d-block cm-s-default border-0">${query['query']}</code></td>
+                        <td>${query['time']}ms</td>
+                        <td>${query['rows']}</td>
+                        <td>${escapeMarkupText(query['warnings'])}</td>
+                        <td>${escapeMarkupText(query['errors'])}</td>
+                    </tr>
+                `;
+            });
+        });
+        sql_table_body.append(rows_to_append);
+
+        if (filtered_request_id !== undefined) {
+            let total_requests = 0;
+            let total_duration = 0;
+            $.each(sql_data['queries'], (request_id, queries) => {
+                if (request_id === filtered_request_id) {
+                    total_requests += queries.length;
+                    queries.forEach((query) => {
+                        total_duration += parseFloat(query['time']);
+                    });
+                }
+            });
+            content_area.find('h1').first()
+                .text(`${_x('debug', '%1$d Queries took %2$dms', 'glpi', total_requests, total_duration)}`);
```

Comment:
Seems there is an issue here, I got something like: `1674$d Queries took 45$dms`